tests/slenkins/login.pm

```diff
@@ -21,7 +21,12 @@ use base 'basetest';
 use testapi;
 
 sub run {
-    select_console('root-console');
+    if (get_var('VIRTIO_CONSOLE')) {
+        select_console('root-virtio-terminal');
+    }
+    else {
+        select_console('root-console');
+    }
```

Comment:
style: how would you like
`select_console(get_var('VIRTIO_CONSOLE') ? 'root-virtio-terminal' : 'root-console');`

Reply:
thx, will change accordingly